daffodil-io/src/main/scala/org/apache/daffodil/processors/charset/IBM037.scala

```diff
@@ -55,3 +55,27 @@ class BitsCharsetDecoderIBM037
     dec
   }
 }
+
```

Comment:
Above. We could define the big table of magic numbers as:
```
  val decodeString = {
    val bytes = ByteBuffer.wrap((0 to 255).map{ _.toByte }.toArray)
    Charset.forName("ebcdic-cp-us").newDecoder().decode(bytes).toString
  }
```
Or we could do this in a separate cleanup pull request also.